### ISSUE TITLE
q&V: hidden locus accessible

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2353]
Not sure if this was purposeful or if it doesn't matter. For features, I get a context list and the hidden locus is visible and clickable: 

<img width="1485" alt="Screenshot 2023-11-05 at 2 01 59 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/89a6ecab-e155-4ced-a629-635f3b488e6d">

<img width="1488" alt="Screenshot 2023-11-05 at 2 03 02 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/7e691519-8c3f-44ea-9055-d3ebd4a51200">

**Comment by luizaogs** [comment]
This is also the case for survey units (look at unit 02, for instance).

**Comment by urapadmin** [comment]
yup. The hidden locus. I could rant about it. That happens if one tries to bend an already failed architecture towards a new goal.
I guess I should supress them here, too. 

Shouldn't we also have different fields for a feature that we currently don't show? I had only focussed on the excavation units (and even there not on legacy units).

**Comment by luizaogs** [comment]
Yeah, if you want to include everything there is stuff missing for both features and survey units.

**Comment by urapadmin** [comment]
- as for missing fields for features and survey units (that nobody ever uses) see #2376 and #2358  and #2643

**Comment by urapadmin** [comment]
### Kiosk 1.6.26
- hidden loci are now hidden from locus lists in Views
